leaf-ui/js/backendComm.js

```diff
@@ -353,3 +338,17 @@ function getIDs(backendErrorMsg) {
 
 	return arr;
 }
+
+function convertToAnalysisResult(results){
```

Comment:
@amgrubb Do you know if it would disrupt the setup if we added this as a class method to the `AnalysisResult` class in `leaf-ui/js/object/Class.js`? Just wondering whether we can streamline a little, but I'm not sure if that would affect communication with the backend at all. Still very shaky on gson

Reply:
Definitely don't think this change would impact anything with gson/json. However, bit confused on purpose of making it a class method in the AnalysisResult object. Currently this code is creating a new AnalysisRequest object from a parsed json file. Its necessary to create a new object to avoid the issue we ran into with last pull request re: overwriting global variables in the config lists. Would you rather we make a new blank AnalysisRequest and then call this method as a class method (maybe renamed to something like convertResults) to populate the values in the new object? I think that could definitely be done, just not clear if that is what you are suggesting.

Reply:
Gotcha - I misunderstood the purpose, I thought it was just wiping the AnalysisRequest object params and repopulating it with new vars, in which case a class func would make more sense. This is fine!